src/plugins/advanced_settings/public/management_app/components/search/search.tsx

```diff
@@ -30,8 +30,8 @@
 
 import React, { Fragment, PureComponent } from 'react';
 import { i18n } from '@osd/i18n';
-import { EuiSearchBar, EuiFormErrorText, Query } from '@elastic/eui';
-
+import { EuiSearchBar, EuiFormErrorText } from '@elastic/eui';
+import { Query } from '@opensearch-project/oui/src/eui_components/search_bar/query';
```

Comment:
This change add a type error in the parent component that uses `Search`. I suggest that you instead add a `ts-ignore` comment to the ts error on line 114. e.g.

```ts
        {/* @ts-ignore The Query types that typescript complains about here are identical and is a false flag. Once OUI migration is complete, this ignore can be removed */}
        <EuiSearchBar box={box} filters={filters} onChange={this.onChange} query={query} />

```

Reply:
@ashwin-pc  That is, do not need to update the import? Will the Query error be fixed in the future?

Reply:
Yeah, we will replace all eui references to oui. That should fix it :)